middlewares/members.js

```diff
@@ -161,6 +161,12 @@ exports.triggerEmailChange = async (req, res) => {
         return errors.makeBadRequestError(res, 'No new email is provided.');
     }
 
+    // eslint-disable-next-line no-useless-escape
+    const EMAIL_REGEX = new RegExp('\@(' + constants.RESTRICTED_EMAILS.join('|').trim() + ')');
+    if (EMAIL_REGEX.test(req.body.new_email)) {
```

Comment:
`if (constants.RESTRICTED_EMAILS.some(email => req.body.new_email.includes(email))` would work better imo

Reply:
not sure if the syntax is right here, but the idea should work

Reply:
Thanks for the suggestion! I've changed it